sim/common/sod/item_effects/phase_5.go

```diff
@@ -161,25 +185,122 @@ func init() {
 		})
 	})
 
-	core.AddEffectsToTest = true
-}
+	// https://www.wowhead.com/classic/item=231784/lightnings-cell
+	//
 
-	// https://www.wowhead.com/classic/spell=465414/crusaders-zeal
-	// Used by:
-	// - https://www.wowhead.com/classic/item=229806/truthbearer and
-	// - https://www.wowhead.com/classic/item=229749/truthbearer
-	func crusadersZealAura465414(character *core.Character) *core.Aura {
-		return character.GetOrRegisterAura(core.Aura{
-			ActionID: core.ActionID{SpellID: 465414},
-			Label:    "Crusaders Zeal (465414)",
-			Duration: time.Second * 8,
+	// Use: Energizes a Paladin with light, increasing melee attack speed by 25%
+	// and spell casting speed by 33% for 20 sec. (2 Min Cooldown)
+	core.NewItemEffect(ScrollsOfBlindingLight, func(agent core.Agent) {
+		character := agent.GetCharacter()
+
+		duration := time.Second * 20
+
+		aura := character.GetOrRegisterAura(core.Aura{
+			Label:    "Blinding Light",
+			ActionID: core.ActionID{SpellID: 467522},
+			Duration: duration,
 			OnGain: func(aura *core.Aura, sim *core.Simulation) {
-				character.PseudoStats.BonusDamage += 15
-				character.MultiplyAttackSpeed(sim, 1.30)
+				character.MultiplyAttackSpeed(sim, 1.25)
+				character.MultiplyCastSpeed(1.33)
 			},
 			OnExpire: func(aura *core.Aura, sim *core.Simulation) {
-				character.PseudoStats.BonusDamage -= 15
-				character.MultiplyAttackSpeed(sim, 1/1.30)
+				character.MultiplyAttackSpeed(sim, 1.0/1.25)
+				character.MultiplyCastSpeed(1.0 / 1.33)
+			},
+		})
+
+		spell := character.GetOrRegisterSpell(core.SpellConfig{
+			ActionID: core.ActionID{ItemID: ScrollsOfBlindingLight},
+			Flags:    core.SpellFlagNoOnCastComplete,
```

Comment:
May want to add the offensive equipment flag to this as well

Reply:
done